opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -66,6 +68,7 @@ def __init__(
         body: Optional[Any] = None,
         resource: Optional[Resource] = None,
         attributes: Optional[Attributes] = None,
+        limits: Optional[SpanLimits] = SpanLimits(),
```

Comment:
Use `SpanLimits` as a reference and add `LogLimits` and use that instead.

Reply:
Done, added tests as well.